modules/remove-books.js

```diff
@@ -0,0 +1,8 @@
+export function removeButton(parentElementNode, btnIndex, rmvBtn, bookData) {
+	rmvBtn.addEventListener('click', () => {
+			bookData.splice(btnIndex, 1);
+			localStorage.setItem('StorageBooks', JSON.stringify(bookData));
+			parentElementNode.remove();
+			window.location.reload();
+	});
+}
```

Comment:
- [ ] According to the requirements, you should refactor your methods and functions to arrow functions. 💯
so kindly refactor `removeButton` to be arrow function 

- [ ] And I think you don't need to reload the page, so I suggest removing this line `window.location.reload();` 

For example:
```
export const removeButton = (parentElementNode, btnIndex, rmvBtn, bookData) => {
  rmvBtn.addEventListener('click', () => {
    bookData.splice(btnIndex, 1);
    localStorage.setItem('StorageBooks', JSON.stringify(bookData));
    parentElementNode.remove();
  });
};
```

![1](https://user-images.githubusercontent.com/45231276/193746751-5c127156-d205-47af-be83-451931e6e5d4.PNG)